lighthouse-viewer/README.md

```diff
@@ -0,0 +1,11 @@
+## Lighthouse Viewer
+
+### Development
+
+* `npm i`
+*  Build: `gulp`, Watch: `gulp watch`
+
+For development, `gulp watch` will browserify `dist/src/main.js` and create a
+runnable script in all modern browsers. Use this for quick iterations when developing.
+
+For production, run `gulp`. This compiles and minifies `dist/src/main.js` using Closure.
```

Comment:
it would be good to add deployment instructions similar to [the extension's readme](https://github.com/GoogleChrome/lighthouse/blob/aa6d38b3ea8b87677e3d1b4b7dfc93c78ed19f44/lighthouse-extension/README.md#manual-deploy-to-cws) here